wordpress-mobile/WordPress-iOS
# [Jetpack Remote Installation] Featureflag and Jetpack login screen update

Refs. #10277 

This PR introduces the `jetpackRemoteInstallation` feature flag and adds the 2 buttons in the Jetpack login screen to open the URLs (Terms and Conditions and FAQ).

![Simulator Screen Shot - iPhone X - 2019-03-26 at 13 14 52](https://user-images.githubusercontent.com/912252/55010153-049c4a00-4fe4-11e9-8e14-82a20fd2b87e.png)

![Simulator Screen Shot - iPhone X - 2019-03-26 at 16 11 25](https://user-images.githubusercontent.com/912252/55010692-eaaf3700-4fe4-11e9-8c5f-c909c7f78986.png)


@bummytime I saw you worked on this view controller, so I'm really happy if you can have a look too 😊
@SylvesterWilmott I attached the 2 screenshots above

**To test:**
- Feature flag: Return _true_ on `.jetpackRemoteInstallation` case
- Add a **self-hosted non Jetpack** site
- Select _Stats_ on the site dashboard or click on _Notifications_ tab
- You should see the Jetpack login view controller
- Tap on _terms and conditions_ button to open the webview
- Tap on _Jetpack FAQ_ to open the webview
